bdk-android/lib/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ repositories {
 }
 
 android {
-    compileSdk = 31
+    compileSdk = 33
```

Comment:
Can we update the targetSdk to 31 as well?

Reply:
sure! updated, see line 26 for the `targetSdk = 33` change